org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/NoActionsAvailableAction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2015 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Andy Wu <andy.wu@liferay.com> - initial API and implementation and initial documentation
+ */
+
+package org.eclipse.buildship.ui.view;
+
+import org.eclipse.jface.action.Action;
+
+import org.eclipse.buildship.ui.util.nodeselection.NodeSelection;
+import org.eclipse.buildship.ui.util.nodeselection.SelectionSpecificAction;
+
+/**
+ * show disabled "No Actions Available" when there is no available actions for some node
+ */
+public final class NoActionsAvailableAction extends Action implements SelectionSpecificAction {
+
+    public NoActionsAvailableAction() {
+        setText("No Actions Avaliable");
```

Comment:
Externalize string, like in the other actions.

Reply:
done